torch/nn/modules/module.py

```diff
@@ -1896,6 +1896,10 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         For state dicts without metadata, :attr:`local_metadata` is empty.
         Subclasses can achieve class-specific backward compatible loading using
         the version number at `local_metadata.get("version", None)`.
+        Additionally, :attr:`local_metadata` can also contain the key
+        `assign_to_params_buffers` that indicates whether keys that are meta
+        tensors in the module should be assigned their corresponding tensor
```

Comment:
Not sure why there is a mention to meta Tensor here? They will all be assigned without distinction on the curr device right?